specs/order_spec.rb

```diff
@@ -141,21 +179,42 @@
     end
 
     it "Returns accurate information about the last order" do
-      # TODO: Your test code here!
+      # 100,Amaranth:83.81;Smoked Trout:70.6;Cheddar:5.63,20,pending
+      id = 100
+      products = {
+        "Amaranth" => 83.81,
+        "Smoked Trout" => 70.6,
+        "Cheddar" => 5.63,
+      }
+      customer_id = 20
+      fulfillment_status = :pending
+
+      order = Order.all.last
+
+      # Check that all data was loaded as expected
+      expect(order.id).must_equal id
+      expect(order.products).must_equal products
+      expect(order.customer).must_be_kind_of Customer
+      expect(order.customer.id).must_equal customer_id
+      expect(order.fulfillment_status).must_equal fulfillment_status
     end
   end
 
   describe "Order.find" do
     it "Can find the first order from the CSV" do
-      # TODO: Your test code here!
+      first = Order.find(1)
+      expect(first).must_be_kind_of Order
```

Comment:
You should also verify that the 1st order's fields are all correct, similar with the last order.